src/scripts/utils.ts

```diff
@@ -0,0 +1,8 @@
+function getCurrentPageNumber(): number {
+  const currentPage = document.querySelector('.garage__page_counter');
+  if (!currentPage) return 1;
+  const pageNumber = Number(currentPage.textContent);
+  return pageNumber;
```

Comment:
Local variable 'pageNumber' is redundant
```suggestion
  return Number(currentPage.textContent);
```